python/shotgun_desktop/upgrade_startup.py

```diff
@@ -127,12 +127,12 @@ def upgrade_startup(splash, sgtk, app_bootstrap):
         # know something wrong is going on.
         logger.exception("Unexpected error when updating startup code.")
         DesktopMessageBox.critical(
-            "Shotgun Desktop update failed",
-            "There is a new update of the Shotgun Desktop, but it couldn't be installed. Shotgun "
+            "ShotGrid Desktop update failed",
+            "There is a new update of the ShotGrid Desktop, but it couldn't be installed. ShotGrid "
             "Desktop will be launched with the currently installed version of the code.\n"
-            "If this problem persists, please contact Shotgun support at "
-            "support@shotgunsoftware.com.\n"
-            "Error: %s" % str(e),
+            "If this problem persists, please <a href='%s'>contact</a> ShotGrid support.\n"
+            "\n"
+            "Error: %s" % (sgtk.support_url, str(e)),
```

Comment:
Seems like we missed this one back in the day.

Everything below this is tk-core and does not need to be reviewed.

Reply:
also found that the README should be updated